l0_sampling/sketch.cpp

```diff
@@ -61,44 +60,44 @@ vec_t Sketch::query() {
 }
 
 Sketch operator+ (const Sketch &sketch1, const Sketch &sketch2){
```

Comment:
Why don't we need to check seed is the same?

Reply:
`operator+` calls `operator+=`, `operator+=` checks that seed is the same. I changed this because felt redundant to have two copies of the same code

Reply:
I see, that makes sense. As a followup, do we really need `operator+`? I don't think we use it anywhere in prod

Reply:
I guess we don't, should i remove it?

Reply:
In that case yeah.